### ISSUE TITLE
Blank Site over Cloudflare Tunnel, Broken Site when connecting locally

### ISSUE THREAD

**Issue by HuckleberryLovesYou** [alexjustesen/speedtest-tracker#1950]
**Describe the bug**
I'm using a Cloudflared instance in a docker container to access speedtest-tracker over a Cloudflare tunnel secured with access. Everything is great if not using Cloudflare.
The problem now is, that if I add `APP_URL` and `ASSET_URL` to the compose.yaml file, and then try to access it via https://speedtesttracker.timmatheis.dev, I get a blank site with two requests returning status code 404:
GET https://speedtesttracker.timmatheis.dev/
GET https://speedtesttracker.timmatheis.dev/favicon.ico

If I then try to access the site over my local network via IP and Port, I get a completely broken website (which allows a bit of interactivity) like shown in the screenshots. Following the console, I see it tries to authenticate with Cloudflare, as well as loading scripts from https://speedtesttracker.timmatheis.dev/ and throws about 30 errors stating:
```
Uncaught TypeError: $store.sidebar is undefined
    anonymous https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710 line 5 > AsyncFunction:3
    Bl https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    Il https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:1
    <anonymous> https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    r https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    $u https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    Te https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:1
    Al https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:1
    <anonymous> https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    r https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    n https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    jl https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    ie https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    Kl https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    Kl https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:5
    za https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:85
    <anonymous> https://192.168.2.148:8443/livewire/livewire.min.js?id=02b08710:86
```

Any other service published via Cloudflare tunnels on the same device work.
For Cloudflare Configuration, take a look at the screenshots.

**To Reproduce**
compose.yaml
```
services:
    speedtest-tracker:
        image: lscr.io/linuxserver/speedtest-tracker:latest
        restart: unless-stopped
        container_name: speedtest-tracker
        ports:
            - 8443:443
        environment:
            - PUID=1000
            - PGID=1000
            - APP_KEY=base64:******
            - DB_CONNECTION=sqlite
            - DISPLAY_TIMEZONE=Europe/Berlin
            - SPEEDTEST_SCHEDULE=4,19,34,49 * * * *
            - SPEEDTEST_SERVERS=*****
            - PRUNE_RESULTS_OLDER_THAN=365
            - TELEGRAM_BOT_TOKEN=******
            - APP_URL=https://speedtesttracker.timmatheis.dev # Change this to your domain name
            - ASSET_URL=https://speedtesttracker.timmatheis.dev # Change this to your domain name
        volumes:
            - /root/docker-data/speedtesttracker:/config

```

**Expected behavior**
Not serving a blank site and show non-broken when connecting locally

**Environment (please complete the following information):**
 - OS: latest Raspberry Pi OS (based on Bookworm), running docker-ce
-  Architecture: arm64
 - Browser Google Chrome, Zen Browser (Firefox fork)
 - Version v1.0.2

**Screenshots**
![image](https://github.com/user-attachments/assets/adc0ecb7-3e2f-4508-8aee-d034aa0b5d50)
![image](https://github.com/user-attachments/assets/9dd284a7-2c89-41a5-a2ed-a3529f920e7d)
![image](https://github.com/user-attachments/assets/a544799e-4d4d-480c-a510-7618b565ffa2)

**Logs**
Nothing in the logs as far as I can see.

**Additional context**
If I access it over Cloudflare in incognito mode, I get redirected successfully to my authentication provider. This means, it shouldn't be Cloudflare Access breaking things. But, since I wasn't 100% sure, I turned off Cloudflare Access for this Tunnel and behavior didn't change.

I've also tried using env. variable FORCE_HTTPS=true, but behavior didn't change.

Best regards

Tim Matheis

**Comment by alexjustesen** [comment]
Can you remove `ASSET_URL` and restart your container? I'm using CF tunnels as well and don't have that set.

**Comment by HuckleberryLovesYou** [comment]
Thanks for reaching out!

I've thought I already tried removing `ASSET_URL`.
Removing `ASSET_URL` fixes the Website when connecting locally, but doesn't affect when connecting to https://speedtesttracker.timmatheis.dev/ over Cloudflare tunnel.

Out of curiosity, I changed the IP on Cloudflare dashboard to something I know wasn't used on my Network. Well, same result. A blank page (not the 'Host not working' Cloudflare page). Speedtesttracker was working totally fine all the time. It seems like the Cloudflare tunnel connector didn't work. I had already tried redeploying it, but it didn't change the behavior. Creating a new Connector resolved this issue for me.

**I'm sorry** for opening this issue and wasting your time.
Still, **thank you very much** for trying to help me.

Best regards

Tim Matheis
